test_util/launch_configs/aws-linux-cluster.yaml

```diff
@@ -0,0 +1,13 @@
+---
+deployment_name: dcos-stack-that-was-provisioned-from-vpc
+deployment_type: linux_cluster
+provider: aws
+provider_info:
+    region: us-west-2
+    access_key_id: ALSAFLAJ4L20922KF20TL294
+    secret_access_key: ADWOFIAOFWJTPW3924JR
+key_helper: true
+linux_cluster:
```

Comment:
If I have 5 t2.micro cent-os-7 provisioned in my AWS account How will this pick 3 out of 5?

The confusing part for me is `dcos-stack-that-was-provisioned-from-vpc` vs providing the details on `os-name`, `instance-type` and `instance_count` below.

Also, what is assumed of the ssh user and password that is required to login to the cluster?